projects/plugins/jetpack/tests/php/modules/masterbar/test-class-wpcom-admin-menu.php

```diff
@@ -345,4 +345,18 @@ public function test_add_gutenberg_menus() {
 		);
 		$this->assertSame( $menu[61], $fse_menu );
 	}
+
+	/**
+	 * Tests add_plugins_menu
+	 *
+	 * @covers ::add_plugins_menu
+	 */
+	public function test_add_plugins_menu() {
+		global $menu;
+
+		static::$admin_menu->add_plugins_menu( true );
```

Comment:
Do we need an assertion to check that when `add_plugins_menu` is given `false`?

Reply:
That's already covered by the default tests done in the `Test_Admin_Menu` class. Here we're only testing the implementation differences that are done in the menu for Simple sites.